x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -282,6 +284,23 @@ export class ExecutionHandler<
     return executables;
   }
 
+  private buildRuleUrl(): string | undefined {
+    if (!this.taskRunnerContext.kibanaBaseUrl || !isString(this.taskInstance.params.alertId)) {
+      return;
+    }
+
+    try {
+      const ruleUrl = new URL(
+        `${triggersActionsRoute}${getRuleDetailsRoute(this.taskInstance.params.alertId)}`,
+        this.taskRunnerContext.kibanaBaseUrl
+      );
+
+      return ruleUrl.toString();
+    } catch (error) {
+      return;
```

Comment:
feels worthy of a `log.debug()`, just in case this case gets caught, but we're confused why the url didn't get set